src/Compilers/CSharp/Portable/BoundTree/GenerateNodes.bat

```diff
@@ -1,7 +1,7 @@
 ﻿REM Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 REM Run the node generator to create new bound tree node definitions.
-REM You must run this in the directory containing Generated.cs.
```

Comment:
I think we should remove this file.
I've been using `build\scripts\generate-compiler-code.cmd`. That one updates all the generated files at once.